index.html

```diff
@@ -33,7 +33,15 @@ <h1>Welcome to the student showcase!</h1>
         <div>THIRTEEN</div>
         <div>FOURTEEN</div>
         <div>FIFTEEN</div>
-        <div>SIXTEEN</div>        
+        <div>Jim Guccione
+                <img src="https://www.google.com/search?q=jim+guccione&source=lnms&tbm=isch&sa=X&ved=0ahUKEwjRsPfvs4TdAhUByYMKHe2QA1UQ_AUICygC&biw=1280&bih=703#imgrc=X_tVn-Uqo0FNDM:" alt ="Jim Guccione">
```

Comment:
Is this image `src` going to resolve to an individual image file? It looks like a URL to search results